kairon/shared/data/processor.py

```
@@ -254,6 +254,211 @@
+            if MongoProcessor.is_slot_in_field(note, slot):
+                return True
+
+        return False
+
+    @staticmethod
+    def is_slot_in_web_search_action_config(config: dict, slot: Text):
+        set_slot = config.get("set_slot")
+        if set_slot == slot:
+            return True
+
+        return False
```

Ruff

Return the condition `set_slot == slot` directly

Replace with `return set_slot == slot`

(SIM103)